Sources/StevensonCore/JiraService.swift

```diff
@@ -0,0 +1,105 @@
+import Foundation
+import Vapor
+
+public struct JiraService {
+    public let baseUrl: String
+    private let headers: HTTPHeaders
+
+    public init(baseUrl: String, username: String, password: String) {
+        self.baseUrl = "\(baseUrl):443"
```

Comment:
port should be part of baseUrl, similarly to `GitHubService`

Reply:
That's what I did at first but then I will likely need to use the base url (without 443 port) in other places — like if one day we figure out our to use URL fields with that damn JIRA API to fill the JIRA url field

Reply:
you can always extract it later I think. it's just that hardcoding port makes it less reusable.